playground/.gitignore

```diff
@@ -0,0 +1,3 @@
+!*
```

Comment:
why need this?

Reply:
Yes that was not needed. Thanks for noticing!